### ISSUE TITLE
Add JSON output to pdbdump

### ISSUE THREAD

**Issue by pombredanne** [jasonwhite/pepatch#12]
I would like to run pdbdump and get structured and easily parsable output for use in tracecode-debug, a tool for an upcoming static analysis companion to https://github.com/nexB/tracecode-build

**Comment by jasonwhite** [comment]
Good idea. I originally wrote `pdbdump` so that I could diff two PDBs to help figure out why `ducible` might not be making a PDB bit-for-bit reproducible. JSON output would still be useful for this purpose.

If you or someone else wants to tackle this, I'd be happy to take a pull request for it. As it is now, I don't have the time for it.

**Comment by pombredanne** [comment]
Thanks. I may chip in when the needs becomes high then. Any preference for a JSON library? I would want to reuse existing code for that part.

**Comment by jasonwhite** [comment]
I haven't done much with JSON in C++, so I don't have much of a preference. The only thing is that it should be easy to integrate into the build. A header-only JSON library would be ideal.

With a quick search, I found this, which looks pretty good: https://github.com/nlohmann/json
